### PR TITLE
chore(analytic): send install event on zfs-localpv initialization

### PR DESCRIPTION
**Pull request by Ab-hishek**
Signed-off-by: Abhishek Agarwal <abhishek.agarwal@mayadata.io>

**Why is this PR required? What issue does it fix?:**

**What this PR does?:**
This PR enables zfs-localpv to send Installation and Ping event

**Does this PR require any upgrade changes?:**

**If the changes in this PR are manually verified, list down the scenarios covered::**

**Any additional information for your reviewer? :**
Mention if this PR is part of any design or a continuation of previous PRs

**Checklist:**
- [ ] Fixes #<issue number>
- [x] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [ ] Has the change log section been updated?
- [ ] Commit has unit tests
- [ ] Commit has integration tests
- [ ] (Optional) Are upgrade changes included in this PR? If not, mention the issue/PR to track:
- [ ] (Optional) If documentation changes are required, which issue on https://github.com/openebs/openebs-docs is used to track them:

### CODE REVIEW

File: pkg/driver/controller.go [hunk 1]
```diff
@@ -119,6 +119,11 @@ func (cs *controller) init() error {
 	go cs.k8sNodeInformer.Run(stopCh)
 	go cs.zfsNodeInformer.Run(stopCh)
 
+	if zfs.GoogleAnalyticsEnabled == "true" {
```
Comment (avishnu) [inline pkg/driver/controller.go:122]: Does this send an event everytime the controller starts?

Comment (Ab-hishek) [reply pkg/driver/controller.go:122]: Yup, seems so.

**Review by pawanpraka1 — APPROVED** [review]
looks good.

**Comment by pawanpraka1** [comment]
@Ab-hishek can you please cherry-pick this to v1.9.x branch.

**Pull request merged**